tests/test_io.rs

```diff
@@ -41,7 +41,7 @@ mod test {
             switched_modulus,
             input_size: 1,
             public_circuit: public_circuit.clone(),
-            d: 3,
+            d: 1,
```

Comment:
@enricobottazzi Could you keep using `d=3` to ensure that the implementation works for `d>1`?

Reply:
i've checked with d=3, it works -- I think we can keep d 1 for now which memory profile for d=3 atm is too heavy

Reply:
Sorry, I forgot to set it back to `d=3`. Btw, the runtime and memory consumption analysis that I reported above is for `d=3`